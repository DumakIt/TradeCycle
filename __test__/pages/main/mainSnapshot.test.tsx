import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import MainPage from "../../../pages";

it("main 페이지 snapshot 테스트", () => {
  const result = render(<MainPage />);
  expect(result.container).toMatchSnapshot();
});
