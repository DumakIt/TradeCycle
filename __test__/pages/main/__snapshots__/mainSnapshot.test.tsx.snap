// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main 페이지 snapshot 테스트 1`] = `
<div>
  <div
    class="css-1g8wo0y"
  >
    <div
      class="css-1axgj1d"
    >
      <div>
        <h1
          class="css-1q6yft4"
        >
          버리지 말아요
        </h1>
        <p
          class="css-byu2th"
        >
          더이상 사용하지 않는 물건 버리거나
          <br />
           그냥 쌓아 두고만 있나요?
        </p>
      </div>
      <img
        class="css-peszeb"
        src="/images/mainImgFirst.webp"
      />
    </div>
    <div
      class="css-1m38szu"
    >
      <img
        class="css-peszeb"
        src="/images/mainImgSecond.webp"
      />
      <div>
        <h1
          class="css-1q6yft4"
        >
          누군가는 필요해요
        </h1>
        <p
          class="css-byu2th"
        >
          나는 필요없어도
          <br />
           누군가는 그것이 필요하답니다
        </p>
      </div>
    </div>
    <div
      class="css-1axgj1d"
    >
      <div>
        <h1
          class="css-1q6yft4"
        >
          여기서 해보아요
        </h1>
        <a
          class="css-15tufge"
          data-testid="btn-list"
        >
          TradeCycle 매물보기
        </a>
      </div>
      <img
        class="css-1kk7qac"
        src="/images/mainImgThird.webp"
      />
    </div>
  </div>
</div>
`;
